torchdynamo/variables/tensor.py

```diff
@@ -392,15 +392,18 @@ def var_getattr(self, tx, name):
 
         return result
 
-    def unpack_var_sequence(self, tx):
+    def unpack_var_sequence_range(self, tx, range):
```

Comment:
Hmm I wonder if we eventually need to do this for all unpacks? As in, make them all take range or provide a default?